examples/data-objects/multiview/container/src/container.tsx

```diff
@@ -109,9 +112,11 @@ export class CoordinateContainerRuntimeFactory extends BaseContainerRuntimeFacto
         triangleCoordinate3.y = 60;
 
         // Create the constellation component
-        const constellationComponent = await requestFluidObject<Constellation>(
-            await runtime.createRootDataStore(Constellation.ComponentName, constellationComponentId),
-            "/");
+        const dataStore = await runtime.createDataStore(Constellation.ComponentName);
```

Comment:
I think ComponentName should be renamed to componentType, and constellationComponentId should be renamed to something containing word "name".

Our samples should be easy to read and convey idea clearly

Reply:
Next example is better here - it uses TodoFactory.type, which is correct.
Here, we can use Constellation.factory().type.
ComponentName should become private and be renamed.